kv/kvserver: TestSplitBurstWithSlowFollower failed

kv/kvserver.TestSplitBurstWithSlowFollower [failed](https://tanzanite.cluster.engflow.com/invocations/default/1e9190f3-1b92-4846-bc41-b5e1c9970ccc?testReportRun=2&testReportShard=40&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on master @ [bd6289d21debf18c3bc0f2889bfe98646c60d1d0](https://github.com/cockroachdb/cockroach/commits/bd6289d21debf18c3bc0f2889bfe98646c60d1d0):


```
=== RUN   TestSplitBurstWithSlowFollower
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestSplitBurstWithSlowFollower2104482723
    test_log_scope.go:76: use -show-logs to present logs inline
    client_split_burst_test.go:198: -- test log scope end --
test logs left over in: outputs.zip/logTestSplitBurstWithSlowFollower2104482723
--- FAIL: TestSplitBurstWithSlowFollower (115.70s)
=== RUN   TestSplitBurstWithSlowFollower/backward
    client_split_burst_test.go:178: 
        	Error Trace:	pkg/kv/kvserver/client_split_burst_test.go:178
        	Error:      	Should be zero, but was 1
        	Test:       	TestSplitBurstWithSlowFollower/backward
--- FAIL: TestSplitBurstWithSlowFollower/backward (104.70s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=40</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSplitBurstWithSlowFollower.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-48385

kv/kvserver.TestSplitBurstWithSlowFollower [failed](https://tanzanite.cluster.engflow.com/invocations/default/b140e9be-8d83-4eac-9470-571083ddc264?testReportRun=2&testReportShard=40&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on master @ [104eff4b76905adeb7de221fbce8a8b2bca7460f](https://github.com/cockroachdb/cockroach/commits/104eff4b76905adeb7de221fbce8a8b2bca7460f):


```
=== RUN   TestSplitBurstWithSlowFollower
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestSplitBurstWithSlowFollower681189760
    test_log_scope.go:76: use -show-logs to present logs inline
    client_split_burst_test.go:198: -- test log scope end --
test logs left over in: outputs.zip/logTestSplitBurstWithSlowFollower681189760
--- FAIL: TestSplitBurstWithSlowFollower (113.17s)
=== RUN   TestSplitBurstWithSlowFollower/backward
    client_split_burst_test.go:178: 
        	Error Trace:	pkg/kv/kvserver/client_split_burst_test.go:178
        	Error:      	Should be zero, but was 1
        	Test:       	TestSplitBurstWithSlowFollower/backward
--- FAIL: TestSplitBurstWithSlowFollower/backward (103.59s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=40</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSplitBurstWithSlowFollower.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/kv/kvserver/client_split_burst_test.go
+++ b/pkg/kv/kvserver/client_split_burst_test.go
@@ -160,9 +160,9 @@ func TestSplitBurstWithSlowFollower(t *testing.T) {
 		// even though there is a slow follower, `splitDelayHelper` isn't interested in
 		// delaying this here (which would imply that it's trying to check that every-
 		// one is "caught up").
-		// We set a 100s timeout so that below we can assert that `splitDelayHelper`
+		// We set a 200s timeout so that below we can assert that `splitDelayHelper`
 		// isn't somehow forcing us to wait here.
-		infiniteDelay := 100 * time.Second
+		infiniteDelay := 200 * time.Second
 		sbt := setupSplitBurstTest(t, infiniteDelay)
 		defer sbt.Stopper().Stop(ctx)
 

